operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-single-vm-view.yaml

```
@@ -0,0 +1,1773 @@
+apiVersion: v1
+data:
+  acm-openshift-virtualization-single-vm-view.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "description": "This dashboard provides detailed insights into a specific virtual machine, including its configuration, resource consumption, and any active alerts. It allows users to easily locate the virtual machine across multiple clusters, facilitating efficient monitoring and troubleshooting.",
```

Similar comment about using VMs. We just need to pick one and stick with it. Either spell it out completely each time or spell it out once and then use the abbreviation
```suggestion
      "description": "This dashboard provides detailed insights of a specific virtual machine including its configuration, resource consumption, and any active alerts. It allows you to easily locate the virtual machine across multiple clusters, facilitating efficient monitoring and troubleshooting.",
```